src/components/TodoList.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useContext } from 'react';
+import { TodoItem } from './TodoItem';
+import { TodosContext } from './TodosContext';
+
+export const TodoList: React.FC = () => {
+  const { filteredTodos } = useContext(TodosContext);
+
+  return (
+    <>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings
```suggestion
    <ul>
```